universal-application-tool-0.0.1/app/views/ViewUtils.java

```diff
@@ -0,0 +1,36 @@
+package views;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.input;
+import static j2html.TagCreator.script;
+
+import controllers.AssetsFinder;
+import j2html.tags.Tag;
+import javax.inject.Inject;
+import play.libs.crypto.DefaultCSRFTokenSigner;
+
+/** Utility class for accessing stateful view dependenices. */
+final class ViewUtils {
+
+  private final DefaultCSRFTokenSigner tokenSigner;
+  private final AssetsFinder assetsFinder;
+
+  @Inject
+  ViewUtils(DefaultCSRFTokenSigner tokenSigner, AssetsFinder assetsFinder) {
+    this.tokenSigner = checkNotNull(tokenSigner);
+    this.assetsFinder = checkNotNull(assetsFinder);
+  }
+
+  /** Generates a hidden HTML input tag containing a signed CSRF token. */
```

Comment:
suggestion (non-blocking): Add to this docstring that this `Tag` should be included in every form.

Reply:
done